files/mods.json

```diff
@@ -408,7 +403,7 @@
         "enabled": true,
         "creator": "W-OVERFLOW",
         "discordcode": "woverflow",
-        "description": "Stop crashes from closing your game and provide solutions!",
+        "description": "Stop crashes from closing your game and (mostly) provides solutions!",
```

Comment:
"and tries to provide a solution" might be better?

Reply:
works for me